src/app/containers/Video/mediatorUrls.js

```diff
@@ -0,0 +1,10 @@
+const mediatorURL = env => {
```

Comment:
1. This could be
```js
const mediatorURL = env =>
  env === 'test' || env === 'live' ? 'open.test.bbc.co.uk' : 'open.bbc.co.uk';
```

2. Filename should not be pluralised
3. This file + its respective test should be inside the `helpers` folder